lib/praxis/restful_doc_generator.rb

```diff
@@ -1,4 +1,26 @@
 module Praxis
```

Comment:
I think any changes to this file should be reverted...we should leave this old one as it is, functioning as it was